src/362a0_len_2f70.c

```diff
@@ -67,7 +67,20 @@ INCLUDE_ASM(void, "362a0_len_2f70", load_hit_data, s32 idx, HitAsset* hit);
 
 INCLUDE_ASM(void, "362a0_len_2f70", parent_collider_to_model, s32 colliderID, s16 modelIndex);
 
-INCLUDE_ASM(s32, "362a0_len_2f70", _add_hit_vert_to_buffer);
+void _add_hit_vert_to_buffer(f32** buf, f32* vert, s32* bufSize) {
+    s32 i;
+
+    for (i = 0; i < *bufSize; i++) {
+        if (buf[i] == vert) {
+            break;
```

Comment:
Interesting... would this match as `return`? It seems logically consistent with what the below `if` does (check if the loop terminated early)

Reply:
If you mean changing it to a return and then the if below is optimization or something, then no that doesn't work